package/scripts/params.py

```diff
@@ -71,7 +71,7 @@
 tmp_kafka_hosts = ''
 for i, val in enumerate(kafka_hosts):
   tmp_kafka_hosts += val + ':' + kafka_bind_port
-  if (i + 1) < len(tmp_kafka_hosts):
+  if i < len(tmp_kafka_hosts):
```

Comment:
Yeah, probably... as you can see, this was originally just a duplicate of zookeeper above it. I'll clean it up in the next iteration.